testing/tests/DevExpress.core/class.tests.js

```diff
@@ -1,4 +1,4 @@
-var Class = require("core/class");
+import Class from 'core/class';
```

Comment:
I suggest to use double quotes instead of single because all quotes in this file are double

Reply:
Same issue with quotes in other changed files